CHANGELOG.md

```diff
@@ -6,8 +6,18 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 <!-- next-header -->
+
 ## [Unreleased] - ReleaseDate
 
+## [0.5.0] - 2025-01-10
+
+### Changed
+
+- BREAKING (#39): Pressing the "left" or "h" keys now folds the current section
```

Comment:
nit: I'd been deliberately not wrapping any lines, and leaving one item per line. Can you undo the word-wrapping changes? (Were those deliberate, or the result of an auto-formatter?)

We could potentially change the formatting here, but I personally prefer to have one's editor do word-wrapping, and we rely on this configuration to merge conflicting `CHANGELOG.md` entries (in principle):

https://github.com/arxanas/scm-record/blob/a8e3340777e64fa0aa9290a73e4175afa6274d08/.gitattributes#L2

Reply:
Sorry about that! My editor automatically did this without me realizing it, and I didn't double check the diff before pushing the commit. Fixed now.